dev/com.ibm.ws.session.cache_fat/bootstrapping.properties

```diff
@@ -0,0 +1,11 @@
+#Auto-generated properties by componenttest.common.apiservices.Bootstrap
+#Tue Sep 22 10:39:43 CDT 2020
+hostName=localhost
+junit_jar=../../ant_build/lib
+HOSTNAME.password=PASSWORD
+localhost.JavaHome=/Library/Java/JavaVirtualMachines/jdk1.8.0_172.jdk/Contents/Home/jre
```

Comment:
This path has to be wrong for `localhost.JavaHome`.  Looks like it points to some Java install on your Mac.

Reply:
Will remove this file

Reply:
done!